.github/ISSUE_TEMPLATE/deployment_bug_report.md

```diff
@@ -20,12 +20,24 @@ Before reporting a problem please check the following. Replace the empty checkbo
 
 ## :computer: System information
 
-- **Operating System:** <!-- Which operating system are you using: Windows, Linux, macOS etc. -->
-- **Data Safe Haven version:** <!-- Which version of the Data Safe Haven are you using: 3.0.0, 3.1.1 etc. -->
+- **Operating System:** <!-- Which operating system you are using: Windows, Linux, macOS etc. -->
+- **Data Safe Haven version:** <!-- Which version of the Data Safe Haven you are using: 3.0.0, 3.1.1 etc. If this is on a branch use `branch name @ 7-letter commit-id` -->
 
-## :cactus: Powershell module versions
+## :cactus: Package dependencies
 
-<!-- Paste the output of deployment/CheckRequirements.ps1 here -->
+<details>
+<summary>List of packages</summary>
+
+<!--
+Powershell: Run `deployment/CheckRequirements.ps1` and paste the output below
+Python: Run `pip freeze` and paste the output below
```

Comment:
```suggestion
Python: Run `hatch run pip freeze` and paste the output below
```

If people are following our instructions that might be better advice.

Reply:
Done in 5133ea6ee